### PR TITLE
ND-427 tfvars from secrets manager

### PR DESCRIPTION
**Pull request by smjmoj**

### CODE REVIEW

**Review by jamesgreen-moj — APPROVED** [review]
LGTM: Would it not be better to pull secrets via terraform data block rather than bash script?

**Review by asifamirat00 — APPROVED** [review]
LGTM

File: .github/workflows/terraform-ci.yaml [hunk 1]
```diff
@@ -58,6 +42,47 @@ jobs:
...
+          TF_VAR_SEONDARY_INTERNAL_CIDR, staff-infrastructure-certificate-services/production/secondary_internal_cidr
+          TF_VAR_PCX_PREPRODUCTION_PKI_OST_CIDR, staff-infrastructure-certificate-services/production/pcx_preproduction_pki_ost_cidr
+          TF_VAR_PCX_PREPRODUCTION_PKI_OST_ID, staff-infrastructure-certificate-services/production/pcx_preproduction_pki_ost_id
+          TF_VAR_PRIMARY_REMOTE_DESTINATION_CIDR, staff-infrastructure-certificate-services/production/primary_remote_destination_cidr
+          TF_VAR_ALZ_CIDR_BLOCK, staff-infrastructure-certificate-services/production/alz_cidr_block
+          TF_VAR_PCX_PRODUCTION_PKI_OST_ID, staff-infrastructure-certificate-services/production/pcx_production_pki_ost_id
+          TF_VAR_SECONDARY_REMOTE_DESTINATION_CIDR, staff-infrastructure-certificate-services/production/secondary_remote_destination_cidr
+          TF_VAR_PRIMARY_INTERNAL_CIDR, staff-infrastructure-certificate-services/production/primary_internal_cidr
+          TF_VAR_GP_CLIENT_PROD_CIDR_BLOCK, staff-infrastructure-certificate-services/production/gp_client_prod_cidr_block
```
Comment (jamesgreen-moj) [inline .github/workflows/terraform-ci.yaml:60]: why are these secrets hard coded to /production/ ? are they not different per environment?

Comment (smjmoj) [reply .github/workflows/terraform-ci.yaml:60]: No, they are the same.
In fact the pre-production environment is in production.
Yes it breaks all patterns and usual practice.

**Review by jamesgreen-moj — APPROVED** [review]

**Pull request closed without merging**